docs/src/pages/[platform]/connected-components/storage/storagemanager/props.ts

```diff
@@ -106,6 +106,12 @@ export const STORAGE_MANAGER = [
     description: 'The heading above the list of files',
     type: `React.ComponentType<FileListHeaderProps>`,
   },
+  {
+    name: `ref?`,
+    description:
+      'The ref for the imperative handle exposed by Storage Manager for manipulating the files state.',
```

Comment:
Think we can condense a bit to something like:
```suggestion
      'Forward ref prop exposing StorageManager imperative methods.',
```

Thoughts?